### PR TITLE
Cyborg beakers and grippers

### PR DESCRIPTION
**Pull request by NanakoAC**
Removed large beakers from crisis and research borgs, and replaced with a new Chemistry Gripper

Plus a couple of little tweaks, fixes and QoL improvements for cyborg grippers. 

changes: 
"Added a new Chemistry Gripper for cyborgs. It holds beakers, bottles, pills, pillbottles, spraybottles, labellers, and phoron sheets"
"Removed Large Beaker from crisis and research cyborgs, replaced with chemistry gripper"
 "Cyborgs can now interact with reagent grinders"
 "Cyborgs can now place objects in their gripper, on a table"
"Cyborgs can now place objects from their gripper into a disposal bin"
"Cyborgs can now use a gripper to take valid objects out of cardboard boxes"
"Cyborgs can now use their other tools, on things held in their gripper"
"Using an empty gripper on a machine now interacts with it"

### CODE REVIEW

**Comment by NanakoAC** [comment]
This fixes #335

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 1]
```diff
@@ -642,7 +641,7 @@
 
 	if(!sheet_reagents[O.type] && (!O.reagents || !O.reagents.total_volume))
 		user << "\The [O] is not suitable for blending."
-		return 1
+		return 0
```
Comment (skull132) [inline code/modules/reagents/Chemistry-Machinery.dm]: Why did you change this return value?

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 2]
```diff
@@ -204,6 +204,7 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/weapon/bonesetter(src)
 	src.modules += new /obj/item/weapon/circular_saw(src)
 	src.modules += new /obj/item/weapon/surgicaldrill(src)
+	src.modules += new /obj/item/weapon/gripper/chemistry(src)
```
Comment (Arrow768) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: I do not support this for the fact that it blurs the line between our two very different medical borgs.

Right now we have one borg that is specialized on surgery and has got a hypospray.

Then we have a more all-round borg that can´t do surgery, but is equipped with the items to work in the chemistry lab.

I think changing this should be discussed on the forums, as it provides a significant edge to one of the two cyborgs.
(With this change the surgical borg can do pretty much everything the crisis borg can do)

File: code/modules/reagents/Chemistry-Machinery.dm (hunk 1 above)
Comment (NanakoAC) [inline code/modules/reagents/Chemistry-Machinery.dm]: The returned value determines whether or not the interaction is 'finished'. If a 1 is returned, the attack terminates and does not call afterattack, which is where the gripper does most of its work

File: code/modules/mob/living/silicon/robot/robot_modules.dm (hunk 2 above)
Comment (NanakoAC) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: Surgical borg is underplayed and often SSD, i was actually thinking, if only one can have it, that the surgical is more suitable than crisis. one works in medical, one works outside

**Comment by NanakoAC** [comment]
Ok theres not much time until the next patch, and i don't want controversy or debate to hold up merging, so ive taken the gripper away from the surgical borg.

File: Nanako-PR-338.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#################################
+
+# Your name.  
+author: Nanako
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: False
```
Comment (skull132) [inline Nanako-PR-338.yml]: Reeee! I thought I checked all of these.
